spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -450,8 +456,8 @@ public SkipperStreamService skipperStreamDeploymentService(
 
 		@Bean
 		public AppRegistryService appRegistryService(AppRegistrationRepository appRegistrationRepository,
-				DelegatingResourceLoader resourceLoader, MavenProperties mavenProperties) {
-			return new DefaultAppRegistryService(appRegistrationRepository, resourceLoader, mavenProperties);
+				AppResourceCommon appResourceService) {
```

Comment:
Should we actually rename `AppResourceCommon` as `AppResourceService` if it's used as a service?

Reply:
Actually i did just opposite. It use to be called AppResourceService but it seems to have been used mostly as stateful util so @markpollack suggested to common as more descriptive term. I will rename the argument names